WalletWasabi.Tests/Helpers/WabiSabiFactory.cs

```diff
@@ -290,39 +291,39 @@ public static OutputRegistrationRequest CreateOutputRegistrationRequest(Round? r
 
 			try
 			{
-				weight ??= Constants.WitnessScaleFactor * script.EstimateOutputVsize();
+				vsize ??= script.EstimateOutputVsize();
 			}
 			catch (NotImplementedException)
 			{
-				weight = 100;
+				vsize = 100; // ??
```

Comment:
What is this?

Reply:
Oh, very good question, I forgot about that! Why is this as it is? What should we do in this case?

Reply:
it seems it is not very important.

Reply:
ACK